changelog.d/20240611_174413_hina.khadim_fix_nightly.md

```diff
@@ -0,0 +1 @@
+- [BugFix] Add variable.scss file path of edx-platform for CMS and remove header and  footer of profile and account MFE   (by @hinakhadim) 
```

Comment:
please update the changelog as we are no longer removing the header/footer in the PR.

Reply:
Current nightly branch has header and footer. We have to remove them in nightly

Reply:
Ah, got it. I was remembering it when they were removed previously, I did not see they have been added back.

Reply:
Because `master` is merged into `nightly`.